infra/docker/keycloak/entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+set -e
+
+sed -i "s;dashboard_url;$DASHBOARD_CLIENT_URL;g" /opt/keycloak/themes/phrasea/email/html/password-reset.ftl
```

Comment:
Try to use a more complex placeholder

```suggestion
sed -i "s;%dashboard_url%;$DASHBOARD_CLIENT_URL;g" /opt/keycloak/themes/phrasea/email/html/password-reset.ftl
```